### ISSUE TITLE
开始转换声音会闪退

### ISSUE THREAD

**Issue by kirari3** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#234]
![B@6~L69UM8G58N)@V`P%6`I](https://user-images.githubusercontent.com/132678909/236473457-654ca16f-4a18-42e4-ad33-d5c1cb2c83cd.jpg)
使用群内提供的模型后出现这个错误 请问这要如何解决 谢谢作者大大们

**Comment by kirari3** [comment]
本人使用的是16系显卡（1660ti）不知道是否和这个有关 我训练的时候也没有index和pth这两个文件 但0428版本不是已经解锁了16系显卡嘛

**Comment by RVC-Boss** [comment]
选错模型了，要选weights文件夹下60MB的那个

**Comment by kirari3** [comment]
> 

你好 我选择是群内提供的模型 依旧会有这种情况 我也询问过其他使用相同模型的群友 他们并没有任何闪退的情况

**Comment by kirari3** [comment]
> 选错模型了，要选weights文件夹下60MB的那个

此外 我刚刚自己训练了一个模型 这次weight里面有pth文件了 但是选择了之后依然有闪退的情况
![image](https://user-images.githubusercontent.com/132678909/236600583-fb166d10-fb98-4364-ae78-4d44628a67ff.png)


以下是我的训练日志
2023-05-06 12:42:45,951	haha2	INFO	{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 3, 'fp16_run': False, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs\\haha2/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs\\haha2', 'experiment_dir': './logs\\haha2', 'save_every_epoch': 5, 'name': 'haha2', 'total_epoch': 20, 'pretrainG': 'pretrained/f0G40k.pth', 'pretrainD': 'pretrained/f0D40k.pth', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 0, 'if_cache_data_in_gpu': 0}
2023-05-06 12:42:45,952	haha2	WARNING	D:\aivc\RVC-beta\train is not a git repository, therefore hash value comparison will be ignored.
2023-05-06 12:42:47,591	haha2	INFO	loaded pretrained pretrained/f0G40k.pth pretrained/f0D40k.pth
2023-05-06 12:43:02,871	haha2	INFO	Train Epoch: 1 [0%]
2023-05-06 12:43:02,876	haha2	INFO	[0, 0.0001]
2023-05-06 12:43:02,876	haha2	INFO	loss_disc=3.229, loss_gen=3.412, loss_fm=10.321,loss_mel=24.430, loss_kl=5.000
2023-05-06 12:43:13,212	haha2	INFO	====> Epoch: 1
2023-05-06 12:43:23,259	haha2	INFO	====> Epoch: 2
2023-05-06 12:43:33,780	haha2	INFO	====> Epoch: 3
2023-05-06 12:43:43,871	haha2	INFO	====> Epoch: 4
2023-05-06 12:43:53,971	haha2	INFO	Saving model and optimizer state at epoch 5 to ./logs\haha2\G_65.pth
2023-05-06 12:43:54,901	haha2	INFO	Saving model and optimizer state at epoch 5 to ./logs\haha2\D_65.pth
2023-05-06 12:44:01,797	haha2	INFO	====> Epoch: 5
2023-05-06 12:44:12,043	haha2	INFO	====> Epoch: 6
2023-05-06 12:44:22,153	haha2	INFO	====> Epoch: 7
2023-05-06 12:44:32,698	haha2	INFO	====> Epoch: 8
2023-05-06 12:44:43,245	haha2	INFO	====> Epoch: 9
2023-05-06 12:44:53,941	haha2	INFO	Saving model and optimizer state at epoch 10 to ./logs\haha2\G_130.pth
2023-05-06 12:44:54,779	haha2	INFO	Saving model and optimizer state at epoch 10 to ./logs\haha2\D_130.pth
2023-05-06 12:45:01,168	haha2	INFO	====> Epoch: 10
2023-05-06 12:45:11,798	haha2	INFO	====> Epoch: 11
2023-05-06 12:45:22,368	haha2	INFO	====> Epoch: 12
2023-05-06 12:45:31,596	haha2	INFO	====> Epoch: 13
2023-05-06 12:45:40,232	haha2	INFO	====> Epoch: 14
2023-05-06 12:45:48,978	haha2	INFO	Saving model and optimizer state at epoch 15 to ./logs\haha2\G_195.pth
2023-05-06 12:45:49,594	haha2	INFO	Saving model and optimizer state at epoch 15 to ./logs\haha2\D_195.pth
2023-05-06 12:45:53,089	haha2	INFO	====> Epoch: 15
2023-05-06 12:45:57,270	haha2	INFO	Train Epoch: 16 [38%]
2023-05-06 12:45:57,274	haha2	INFO	[200, 9.981266397366609e-05]
2023-05-06 12:45:57,275	haha2	INFO	loss_disc=3.388, loss_gen=2.938, loss_fm=7.138,loss_mel=20.546, loss_kl=1.940
2023-05-06 12:46:02,404	haha2	INFO	====> Epoch: 16
2023-05-06 12:46:11,007	haha2	INFO	====> Epoch: 17
2023-05-06 12:46:19,546	haha2	INFO	====> Epoch: 18
2023-05-06 12:46:28,042	haha2	INFO	====> Epoch: 19
2023-05-06 12:46:36,525	haha2	INFO	Saving model and optimizer state at epoch 20 to ./logs\haha2\G_260.pth
2023-05-06 12:46:37,273	haha2	INFO	Saving model and optimizer state at epoch 20 to ./logs\haha2\D_260.pth
2023-05-06 12:46:39,799	haha2	INFO	====> Epoch: 20
2023-05-06 12:46:39,799	haha2	INFO	Training is done. The program is closed.
2023-05-06 12:46:40,054	haha2	INFO	saving final ckpt:Success.

请过目一下 谢谢

**Comment by kirari3** [comment]
![image](https://user-images.githubusercontent.com/132678909/236600991-55b3c1f4-5f12-4d66-9994-5ca3832b879a.png)
软件设置情况

**Comment by Duo1a** [comment]
我也是，一打开“开始音频转换”就卡住了，cmd没有一点变化
